candi/cloud-providers/gcp/openapi/doc-ru-cluster_configuration.yaml

```diff
@@ -72,6 +72,8 @@ apiVersions:
                   > **Внимание!** Сейчас поддерживается и тестируется только `Ubuntu 18.04`, `Ubuntu 20.04`, `Ubuntu 22.04`, `Centos 7`, `Centos 8`, `Centos 9`, `Debian 9`, `Debian 10`, `Debian 11`.
               diskSizeGb:
                 description: Размер root-диска. Значение указывается в `ГиБ`.
+              etcdDiskSizeGb:
+                description: Размер диска для etcd. Значение указывается в `ГиБ`.
```

Comment:
Please check description in different cloud providers - we use gigabytes or gibibytes ?

Reply:
We use gigabytes everywhere except GCP, but in the documentation all providers mention GiB (2^30 bytes). So I changed GB to GiB everywhere.